dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -945,10 +945,16 @@ def sample(self, model: dwave.optimization.Model,
 
         Returns:
             :class:`~concurrent.futures.Future` [SampleResult]:
-                Named tuple containing nonlinear model and timing info, in a Future.
+                Named tuple, in a Future, containing the nonlinear model and general
```

Comment:
```suggestion
                Named tuple, in a ``Future``, containing the nonlinear model and general
```
or maybe we mean "future"?

Reply:
Or could make it a cross-ref to [dwave-cloud-client Future](https://docs.dwavequantum.com/en/latest/ocean/api_ref_cloud/computation.html#dwave.cloud.computation.Future)

Reply:
It's a built-in `Future`, as documented for the return type, two lines above. Should we repeat the ref here?

Reply:
No, ignore my comment